pagespeed/system/redis_cache_test.cc

```diff
@@ -45,6 +45,7 @@ namespace net_instaweb {
 namespace {
   static const int kReconnectionDelayMs = 10;
   static const int kTimeoutUs = 100 * Timer::kMsUs;
+  static const int kDatabaseIndex = 0;
```

Comment:
It's fine to have a constant index for the indexing tests.  But can you add a test that opens 2 clients with the same redis server, different DB indexes.  Make sure a given key in one client is independent of the same key in the other one.